pkg/controller/configmap/configmap_controller.go

```diff
@@ -409,7 +428,11 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 				}
 				r.status.SetNotDegraded(statusmanager.ClusterConfig)
 				r.status.SetNotDegraded(statusmanager.OperatorConfig)
-				return reconcile.Result{}, nil
+
+				if !nsxNodeAgentDsChanged {
```

Comment:
Do you think we should put this in the same level with `ncpDeploymentChanged` at line 418?

Reply:
I think we should put nsxNodeAgentDsChanged check either upper lever or lower level at line 418 since each step is to to do further check to include all conditions. In the end, it check there is no any changes happen.

Reply:
Yes, agreed. So I refactor code a little bit as at line 425:
 if !ncpDeploymentChanged && !nsxNodeAgentDsChanged {